modules/network-observability-loki-install.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-loki-installation_{context}"]
+= Installing the Loki Operator
+
+.Prerequisites
+
+* Installed Loki, recommended to install Loki using the Operator version 5.6, which allows the ability to specify a `network` tenant for Loki. The `network` tenant uses a specific tenant mode only implemented in version 5.6 of the Loki Operator. 
+* Object storage, such as AWS S3
+* {product-title} 4.10+
+* Linux Kernel 4.18+
+
+//* <Any Loki install prerequisites for using with Network Observability operator?>
+
+There are several ways you can install Loki. One way is by installing the Loki Operator using the {product-title} web console Operator Hub.
+
+It is recommended to deploy the Loki Operator in the same namespace as the Network Observability operator. You can use the web console or CLI to create a namespace, or new project:
+
+- Using the web console, navigate to the *Project: All Projects* dropdown and select *Create Project*. Name the project `netobserv` and click *Create*.
+- Using the CLI, run the following command to create the`netobserv` namespace: 
++
+[source, terminal]
+----
+$ oc create ns netobserv
+----
```

Comment:
@stleerh @memodi @skrthomas  maybe I missed a discussion, but is it really what we recommend: to install the *operator* in `netobserv` namespace? And Loki Operator too?

Or is there a confusion between _where the operator is installed_ and _where the corresponding stack is deployed_ ?

While this procedure probably works well, I wouldn't say it's recommended or even desirable (the Loki _operator_ is not tied to NOO ; only the _LokiStack_ instance is)

So to me it would be better to say, roughly: install the two operators (we don't really care where, just use the defaults); then create a `netobserv` project to install the `LokiStack` and use that namespace for `FlowCollector` as well.

The sentence:
> It is recommended to deploy the Loki Operator in the same namespace as the Network Observability operator.

Should be rewritten something like:
> It is recommended to deploy the `LokiStack` in the same namespace referred in `FlowCollector` `spec.namespace`.

Reply:
In fact I'm not even sure we can install the operator itself in another namespace .. I looked at it this morning and I didn't see the option proposed using the console

Reply:
Yes I think this is confusion on my part. In Steven's video I noticed the LokiStack was in the `netobserv` namespace and also mentioned when creating the netobserv project that we would be putting all our operator items in that project. 

Moreover, I think this information about `LokiStack` needs to be relocated to modules/network-observability-lokistack-create.adoc. 
![image](https://user-images.githubusercontent.com/77501157/208941340-91e5c285-a280-4ad5-8cbe-201747e4040d.png)